task1-2.c

```diff
@@ -16,20 +16,20 @@ double geometric_mean(double x, double y);
 int main(){
     double i, j, sr_arifm_cube, sr_geom;
 
-    printf("%s", "In the next strings insert just non-negative or just negative numbers\n");
+    puts("In the next strings insert just non-negative or just negative numbers\n");
 
     scanf("%lf", &i);
     scanf("%lf", &j);
 
-    sr_arifm_cube = Arithmetic_mean_of_cubes(i, j);
+    sr_arifm_cube = arithmetic_mean_of_cubes(i, j);
     sr_geom = geometric_mean(i, j);
```

Comment:
[`geometric_mean`](https://translate.google.com/?hl=ru&sl=ru&tl=en&text=%D1%81%D1%80%D0%B5%D0%B4%D0%BD%D0%B5%D0%B5%20%D0%B3%D0%B5%D0%BE%D0%BC%D0%B5%D1%82%D1%80%D0%B8%D1%87%D0%B5%D1%81%D0%BA%D0%BE%D0%B5&op=translate)